brooklyn-cloudfoundry/src/main/java/org/apache/brooklyn/cloudfoundry/entity/CloudFoundryAppFromManifestImpl.java

```diff
@@ -37,7 +37,7 @@ private void checkConfiguration() {
         String configurationContents = getConfig(CONFIGURATION_CONTENTS);
 
         // Exactly one of the two must have a value
-        if (Strings.isBlank(configurationUrl) == Strings.isBlank(configurationContents))
+        if (Strings.isBlank(configurationUrl) && Strings.isBlank(configurationContents))
```

Comment:
You are right. I did not understand well the Exactly one condition :-) I have reverted this kind of changes. Sorry